lib/actions/deploy.coffee

```diff
@@ -184,6 +185,7 @@ module.exports =
 
 		logger.logDebug('Parsing input...')
 
+		appName = undefined
```

Comment:
This is unnecesary

Reply:
In this case this statement is necessary, as otherwise the coffeescript compiler generates `var appName` in the wrong scope. Without it, the linter says:
```
  ⚡ lib/actions/deploy.coffee
     ⚡ #226: Undefined identifier "appName".
```
And when the CLI is executed I get:
```
ReferenceError: appName is not defined
```

Reply:
Ah the scope is a good point, sorry for the noise.